### PR TITLE
Troubleshooting installation process with observed errors and working solutions

### PR DESCRIPTION
**Pull request by meteozay**
Troubleshooting errors observed during installation and building of NNTrainer.

### CODE REVIEW

File: docs/getting-started.md [hunk 1]
```diff
@@ -144,3 +144,32 @@ gbs build
...
+
+## Troubleshooting
+
+### Error 1:
+
+Applications/TransferLearning/Draw_Classification/jni/meson.build:5:0: ERROR: Dependency "glib-2.0" not found, tried pkgconfig and cmake
+
+### Solution: Please install libglib2.0-dev using the following:
+
+sudo apt-get install libglib2.0-dev
```
Comment (kparichay) [inline docs/getting-started.md:157]: how about enclosing this in bash code block like below:
```bash
sudo apt-get install libglib2.0-dev
```
for this and 2 instructions below as well?

**Review by kparichay — APPROVED** [review]
LGTM other than the minor comment.

File: docs/getting-started.md (hunk 1 above)
Comment (meteozay) [reply docs/getting-started.md:157]: Sure, in the next commit, we can update with the code block.

**Review by jijoongmoon — APPROVED** [review]
LGTM

**Pull request merged**